demos/guide/tls-client-block.c

```diff
@@ -162,7 +172,7 @@ int main(int argc, char *argv[])
      * Create the underlying transport socket/BIO and associate it with the
      * connection.
      */
-    bio = create_socket_bio(hostname, port);
+    bio = create_socket_bio(hostname, port, ipv6 == 0 ? AF_INET : AF_INET6);
```

Comment:
nit: consistency: this one is written differently, change to be the same

Reply:
fixed